### PR TITLE
update upgrade instructions

### PR DESCRIPTION
**Pull request by tsonevn**
update upgrade instructions

### CODE REVIEW

**Review by NickIliev — APPROVED** [review]

File: docs/releases/upgrade-instructions.md [hunk 1]
```diff
@@ -135,24 +137,3 @@ npm i nativescript-angular@latest --save
 ./node_modules/.bin/update-app-ng-deps
 npm i
 ```
-
-## Upgrading Webpack
```
Comment (elena-p) [inline docs/releases/upgrade-instructions.md:139]: We can keep this section (after Upgrading TNS modules) as we explain both `tns update` and the separate steps

File: docs/releases/upgrade-instructions.md [hunk 2]
```diff
@@ -82,7 +82,19 @@ You can also switch to specific version by passing it to the command:
 tns update 2.3.0
 ```
 
-> **Note:** The command `tns update` is updating the `tns-core-modules` and the runtimes (`tns-android` and `tns-ios`). The command is combining the next two commands in this article (`tns platform add` and `npm i tns-core-modules`).
+> **Note:** The command `tns update` is updating the `tns-core-modules`, `nativescript-dev-webpack`, and the runtimes (`tns-android` and `tns-ios`). The command is combining the next two commands in this article (`tns platform add` and `npm i tns-core-modules`).
```
Comment (elena-p) [inline docs/releases/upgrade-instructions.md]:
```suggestion
> **Note:** The command `tns update` is updating the `tns-core-modules`, `nativescript-dev-webpack`, and the runtimes (`tns-android` and `tns-ios`). The command is combining the next three commands in this article (`tns platform add`, `npm i tns-core-modules` and `npm i nativescript-dev-webpack --save-dev`).
```

**Comment by tsonevn** [comment]
test

File: docs/releases/upgrade-instructions.md [hunk 3]
```diff
@@ -82,7 +82,19 @@ You can also switch to specific version by passing it to the command:
...
+
+### Updating Webpack configuration
+
+When upgrading an existing version of the Webpack plugin with `tns update` command, you should consider that the related development dependencies also have to be updated accordingly. To ease the process, the plugin provides an automated script for that purpose called `update-ns-webpack` located in `<project-folder>/node_modules/.bin` folder. The script comes with two flags:
```
Comment (elena-p) [inline docs/releases/upgrade-instructions.md]: Not sure if it is a good idea to have this "When upgrading an existing version of the Webpack plugin with `tns update` command, you should consider that the related development dependencies also have to be updated accordingly. To ease the process, the plugin provides an automated script for that purpose called `update-ns-webpack`" as a second note and link to the section on the page (if we decide to keep both webpack sections)

**Review by elena-p — CHANGES_REQUESTED** [review]

**Review by elena-p — APPROVED** [review]

**Comment by tsonevn** [comment]
test

**Comment by tsonevn** [comment]
test

**Comment by tsonevn** [comment]
test

**Pull request merged**